packages/nimble-components/src/utilities/type-extensions.d.ts

```diff
@@ -0,0 +1,21 @@
+/**
```

Comment:
Can you move this file to `src/global-type-extensions.d.ts`

This isn't really a scoped utility and has a global impact so lets not hide it here.